tests/sentry/event_manager/test_hierarchical_hashes.py

```diff
@@ -202,54 +202,18 @@ def test_partial_move(default_project, fast_save):
 
 
 class EventManagerGroupingTest(TestCase):
-    def test_applies_secondary_grouping_hierarchical(self):
```

Comment:
This is easier to review with the split view
<img width="221" alt="image" src="https://github.com/getsentry/sentry/assets/44410/eae4ebdb-9287-4ca2-a31b-9aaf3d44a484">